android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java

```diff
@@ -19,42 +20,50 @@
         this.mDb = db;
     }
 
-    protected abstract T cursorToEntity(Cursor cursor);
+    protected abstract T cursorToEntity(@NonNull Cursor cursor);
 
-    protected long insert(String tableName, ContentValues values) {
+    @IntRange(from = -1)
+    protected long insert(@Nullable String tableName, @NonNull ContentValues values) {
```

Comment:
`tableName` should probably be `@NonNull`?

Reply:
Oops. Fixed.